src/main/resources/application.yml

```diff
@@ -1 +1,7 @@
-
+spring:
+  datasource:
+    username:
```

Comment:
JFYI: this is a good decision, any sensitive data should not be merged. 
resolve this comment by yourself